Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'scientist', '~> 1.0.0'
 gem 'flipper'
 gem 'flipper-active_record'
 gem 'flipper-ui'
+gem 'panoptes-client', github: "zooniverse/panoptes-client.rb"
```

Comment:
I've usually just been releasing the gem as patchlevel version bumps, as long as it's just adding more supported API calls.